plugins/woocommerce/src/Internal/Admin/CategoryLookup.php

```diff
@@ -65,7 +63,7 @@ public function regenerate() {
 		global $wpdb;
 
 		// Delete existing data and ensure schema is current.
-		Install::create_tables();
+		\WC_Install::create_tables();
```

Comment:
I'm tempted to suggest we just remove this line because calling `create_tables` seems unnecessary at this point. The schema should already be current?

My concern would be if performance degrades because all of the Core base tables would now be processed in this function.

Reply:
@adrianduffell That's a good idea.

I think the reason we called `create_tables` in this callback is to prevent errors in a situation when WCA install process kicks in first and then attempts to access category-related tables. We even create an [event](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Internal/Admin/Install.php#L530) to call `generate` function. I think it's safe to delete it now.

Updated in https://github.com/woocommerce/woocommerce/pull/32513/commits/b37421fb981b121ba8956f46bdd06f69cf4a1358